framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -58,6 +62,9 @@ SideSetsAroundSubdomain::modify()
   if (!_mesh_ptr)
     mooseError("_mesh_ptr must be initialized before calling SideSetsAroundSubdomain::modify()!");
 
+  // Initialize the BlockRestrictable parent
+  initializeBlockRestrictable(_pars);
```

Comment:
I don't like that the Class has to remember to do this itself.

Perhaps just make `MooseApp` do this before `modify()` is run?  If it's already initialized then it will just be skipped, right?